### PR TITLE
[id] added: prepare translation indonesian for 4.3

### PR DESCRIPTION
**Pull request by ddevsr**
Prepare for translation indonesian errors and consistency in `4.3`

### CODE REVIEW

File: Language/id/Errors.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+// Errors language settings
+return [
+    'pageNotFound'    => '404 - Halaman tidak ditemukan',
+    'sorryCannotFind' => 'Maaf! Sepertinya tidak dapat menemukan halaman yang Anda cari.',
+    'whoops'          => 'Upps!',
+    'weHitASnag'      => 'Kami menemukan adanya halangan, Silahkan coba kembali nanti...',
```
Comment (samsonasik) [inline Language/id/Errors.php:17]: Typo: silahkan -> silakan

```suggestion
    'weHitASnag'      => 'Kami menemukan adanya halangan, silakan coba kembali nanti...',
```

**Comment by MGatner** [comment]
Leaving this one to @samsonasik

File: Language/id/Errors.php (hunk 1 above)
Comment (ddevsr) [reply Language/id/Errors.php]: Done

**Review by samsonasik — APPROVED** [review]
Looks good 👍 , Build errors seems unrelated.

**Comment by samsonasik** [comment]
Thank you @ddevsr

**Pull request merged**